tools/manifest/download.py

```diff
@@ -67,7 +62,7 @@ def score_name(name):
     # Accept both ways of naming the manifest asset, even though
     # there's no longer a reason to include the commit sha.
     if name.startswith("MANIFEST-") or name.startswith("MANIFEST."):
-        if zstandard and name.endswith("json.zst"):
```

Comment:
I'd prefer not not change the download code, since this could be used by other entrypoints that don't install zstandard.

Reply:
If I account for all imports of this code, would that help? Or do you mean uses of this code that aren't in the repo?

Reply:
Yeah. I think in practice gecko often has zstandard installed anyway, but "we want to make a compiled dependency mandatory for all users" is RFC material since it could break CI systems. I think it's already an assumption that users who are running `wpt <command>` and using the built in venv are able to install stuff from PyPI, but that doesn't extend to people using a different frontend or `wpt` with a pre-exisiting virtualenv.

Reply:
Alright, I'll just revert all changes to download.py.

Reply:
Except the `run(venv, **kwargs)` bit...